SetupDataPkg/Tools/SettingSupport/UefiVariablesSupportLinuxLib.py

```diff
@@ -2,8 +2,11 @@
 #
 # Python lib to support Reading and writing UEFI variables from Linux
 #
+# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.

```

Comment:
Their license is MIT. Can we merge it into this BSD clause?

Reply:
based on offline conversations, tweaked to handle the case.